### PR TITLE
[pt-br] Update docs/concepts/storage/volumes.md

### LINKED ISSUE #41502: [pt-br] Update content/pt-br/docs/concepts/storage/volumes.md
Update the awsElasticBlockStore volume type by adding the (deprecated) in the Portuguese version.
Please assign it to me :)

### PR DESCRIPTION
**Pull request by AlmeidaJoao**
Fixes #41502 
This PR aims to update the Portuguese version of the volumes page, by marking the deprecated volume types.

### CODE REVIEW

File: content/pt-br/docs/concepts/storage/volumes.md
[the remark this answers is not in the document]
Comment (AlmeidaJoao) [reply content/pt-br/docs/concepts/storage/volumes.md:1]: Yes, definitely!!!

[the remark this answers is not in the document]
Comment (AlmeidaJoao) [reply content/pt-br/docs/concepts/storage/volumes.md:1]: > Hi there, @AlmeidaJoao! 👋
> 
> Thank you very much for your contribution! 🎉
> 
> I've noticed there are some more bits in this page that could be updated. Would you feel comfortable doing so?
> 
> Olá, @AlmeidaJoao! 👋
> 
> Muito obrigada pela tua contribuição! 🎉
> 
> Percebi que algumas outras partes desta página ainda estão desatualizadas. Tu te sentirias confortável em atualizá-las também?

Yes, I can do it!

[the remark this answers is not in the document]
Comment (AlmeidaJoao) [reply content/pt-br/docs/concepts/storage/volumes.md:1]: @stormqueen1990 I have updated some more bits of the page I found outdated.

[the remark this answers is not in the document]
Comment (stormqueen1990) [reply content/pt-br/docs/concepts/storage/volumes.md:1]: Thanks! Feel free to mark this conversation as resolved 😃

File: content/pt-br/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -637,9 +662,10 @@ Como operador do cluster Kubernetes que administra o armazenamento, aqui estão
 
 Um volume `secret` é usado para passar informações sensíveis, tais como senhas, para Pods. Você pode armazenar segredos na API Kubernetes e montá-los como arquivos para serem usados por pods sem necessidade de vinculação direta ao Kubernetes. Volumes `secret` são mantidos pelo tmpfs (um sistema de arquivos com baseado em memória RAM) para que nunca sejam gravados em armazenamento não volátil.
 
-{{< note >}}Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo. {{< /note >}}
-
-{{< note >}} Um contêiner que utiliza um Secret como ponto de montagem para a propriedade [`subPath`](#using-subpath) não receberá atualizações deste Secret. {{< /note >}}
+{{< note >}}
+* Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo. {{< /note >}}
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/storage/volumes.md:666]:
```suggestion
* Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo.
```

It seems an extra closure for the `{{< note >}}` shortcode was left here, and that's causing the build to fail.

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**